src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/select-product-targets-alert-condition.mdx

```diff
@@ -1,37 +1,37 @@
 ---
-title: Select product targets for the alert condition
+title: Select targeted entities for the alert condition
 tags:
   - Alerts and Applied Intelligence
   - New Relic Alerts
   - Alert conditions
-metaDescription: Read about how to create notification channels for your policy conditions.
+metaDescription: Read about how to create, view, and update entities in notification channels for your policy conditions.
 redirects:
   - /docs/alerts/new-relic-alerts-beta/configuring-alert-policies/selecting-product-targets-alert-condition
   - /docs/alerts/new-relic-alerts-beta/configuring-alert-policies/select-product-targets-alert-condition
   - /docs/alerts/new-relic-alerts/configuring-alert-policies/select-product-targets-alert-condition
   - /docs/alerts/new-relic-alerts/defining-conditions/select-product-targets-alert-condition
 ---
 
-As part of the [policy workflow](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/alert-policy-workflow), create a condition for your policy. This involves three steps:
+You can monitor the types of data that matter most to your business using targeted entities. 
 
-1. Select the available [product category](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions) (APM, Browser, Mobile, etc.) and the type of condition (metric or external service).
-2. Select specific entities or components monitored by that product as one or more **targets** for the alert condition.
-3. Define the criteria ([thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-thresholds-trigger-alert)) that trigger a violation incident.
+A targeted entity is a specific entity you can create, view, and update. Learn how you can select and modify targeted entities.
 
-This describes how to add and update the product targets.
+<Callout variant="tip">
+  This document helps set targeted entities for your alert condition. You must create a condition for your policy first before selecting product targets. For creating a condition, see [Create alerts conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/).
```

Comment:
Consider making a heading for "Before you get started" (instead of a tip) OR Consider putting this information into an TIP callout
-Poke around to see how other writers handle `prerequisites`